pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -824,26 +803,6 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
       const defaultConfig = { credsStore: 'rancher-desktop' };
       let existingConfig: Record<string, any>;
 
-      const OldCredHelperService = '/etc/init.d/credhelper-vtunnel-peer';
-      const OldCredHelperConfd = '/etc/conf.d/credhelper-vtunnel-peer';
```

Comment:
Do we need to delete any existing copies of the old services? (Including `/etc/init.d/vtunnel-peer`)

Reply:
I already deleted it under `assets/scripts`

Reply:
Sorry, I should have been clearer: when a user upgrades from an older version of Rancher Desktop, wouldn't the existing files be left behind? Do we need to delete those?

(This would be the same reason we had migrations like this before)